android/capacitor/src/main/java/com/getcapacitor/BridgeWebChromeClient.java

```diff
@@ -48,17 +48,12 @@ public void onShowCustomView(View view, CustomViewCallback callback) {
     callback.onCustomViewHidden();
     super.onShowCustomView(view, callback);
   }
-  
-  @Override
-  public void onHideCustomView() {
-    super.onHideCustomView();
-  }
```

Comment:
this method is need for full screen video button to appear in players

Reply:
This was a change from this PR: https://github.com/ionic-team/capacitor/pull/2917

My understanding is a method that only calls the parent class's method can simply be removed and it will use the parent class's implementation.

Reply:
I only know that if you remove it, the button disappear, if you add it back, the button appears, so despite it only calls the super method, looks like it's needed for the full screen button to appear and work.